asterisk/rootfs/etc/cont-init.d/asterisk.sh

```diff
@@ -27,7 +27,6 @@ fi
 readonly target_certfile="/etc/asterisk/keys/fullchain.pem"
 readonly target_keyfile="/etc/asterisk/keys/privkey.pem"
 
-mkdir -p /etc/asterisk/keys
```

Comment:
I think this is needed

Reply:
Fixed in https://github.com/TECH7Fox/Asterisk-add-on/pull/27/commits/16e22f4d6c761f6c2c19eb603a3ef80c2ff4f692